### DISCUSSION TITLE
Unable to generate mitmproxy-ca-cert.p12 key

### DISCUSSION THREAD

**Discussion by Chosen1One999** [mitmproxy/mitmproxy#6565] [Q&A]
Hello, recently I tried to use a packet capture tool, which requires generating a mitmproxy-ca-cert.p12 key using mitmproxy. However, I struggled to generate it, despite multiple attempts. I've already installed Python 3.1.2 and thought that by installing Python and then running mitmdump through the command prompt with mitmproxy, I'd be able to generate the file I need. Later, I found out that Python was not added to the system environment variables, so I added it, and also deleted the originally generated key folder. Then I used mitmdump again to generate the key, but still, the mitmproxy-ca-cert.p12 that I wanted was not generated. I'm not sure what went wrong.

**Comment by rosydawn6** [comment]
Running mitmdump generates that file for me.  What steps have you followed to install mitmproxy/mitmdump?  Which version of mitmproxy and python are you using?   See https://docs.mitmproxy.org/stable/overview-installation/ for more details.

**Comment by Chosen1One999** [comment]
"The detailed description is in the attachment, thank you for your response."



Galaxy.&nbsp;Chen
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mitmproxy/mitmproxy"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月6日(星期六) 中午1:00
***@***.***&gt;;
抄送:&nbsp;"Galaxy. ***@***.******@***.***&gt;;
主题:&nbsp;Re: [mitmproxy/mitmproxy] Unable to generate mitmproxy-ca-cert.p12 key (Discussion #6565)





 
Running mitmdump generates that file for me.  What steps have you followed to install mitmproxy/mitmdump?  Which version of mitmproxy and python are you using?   See https://docs.mitmproxy.org/stable/overview-installation/ for more details.
